conf/machine/include/imx-base.inc

```diff
@@ -281,6 +281,15 @@ MACHINE_GSTREAMER_1_0_PLUGIN_mx7d   ?= "gstreamer1.0-plugins-imx-meta"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mm  ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mn  ?= "imx-gst1.0-plugin"
 MACHINE_GSTREAMER_1_0_PLUGIN_mx8mp  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8mq  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8qm  ?= "imx-gst1.0-plugin"
+MACHINE_GSTREAMER_1_0_PLUGIN_mx8qxp ?= "imx-gst1.0-plugin"
+
+PREFERRED_VERSION_gstreamer1.0_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-base_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-good_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-plugins-bad_mx8 ?= "1.16.imx"
+PREFERRED_VERSION_gstreamer1.0-libav_mx8 ?= "1.16.0"
```

Comment:
The libav does not have a fork, so we can let it float right?

Reply:
meta-freescale provides gstreamer1.0-libav_1.16.0.bb to be compliant with gstreamer 1.16.imx
poky provides gstreamer1.0-libav_1.16.2.bb to compliant with the stock gstreamer 1.16.2
libav should be set according what choosen for Gstreamer, am I wrong?

Reply:
You're right. In this case, we ought to rename the recipe, so it is clearly a  `.imx` version as the others.

Reply:
It's not a custom NXP/imx version, it's just the stock 1.16.0 version, no longer available in poky due to updates to 1.16.2.
I'm wondering if it make sense remove it completely libav recipes an references
...

Reply:
@varigigi I know it is not a real fork but we could use the specific version to avoid confusion. What do you think?